src/main/java/pl/noip/lolstats/lol/stats/Exceptions/NoNameInTokenException.java

```diff
@@ -0,0 +1,12 @@
+package pl.noip.lolstats.lol.stats.Exceptions;
+
+public class NoNameInTokenException extends JwtException {
```

Comment:
Maybe more generic name, indicating that some claim is missing. Create Enum for claim name and use it as parameter.